--- origsrc/libqalculate-0.9.6/configure.in	2007-05-18 03:08:26.000000000 -0500
+++ src/libqalculate-0.9.6/configure.in	2008-02-04 21:25:21.062500000 -0600
@@ -114,7 +114,7 @@
 USE_READLINE="yes"
 READLINE_LIBS=""
 if test "x$enable_textport" = "xyes" ; then
-  QALCULATE_TEXT="qalc"
+  QALCULATE_TEXT="qalc\$(EXEEXT)"
   LN_QALCULATE="qalc"
 
   AC_MSG_CHECKING([whether to use readline])
@@ -199,7 +199,7 @@
 
 QALCULATE_DEFS2DOC=""
 if test "x$enable_defs2doc" = "xyes" ; then
-	QALCULATE_DEFS2DOC="defs2doc"
+	QALCULATE_DEFS2DOC="defs2doc\$(EXEEXT)"
 fi
 AC_SUBST(QALCULATE_DEFS2DOC)
 
